Fix Template and codequality

mastacheata opened the pull request:
* fix template reference if cache is invalidated
* move codequality report to individual jobs

mastacheata committed:
Update .gitignore

fix gitignore and sort correctly

mastacheata committed:
Merge branch 'ambient-innovation:main' into main

mastacheata committed:
feat: fix template

* fix template reference if cache is invalidated
* move codequality report to individual jobs

mastacheata committed:
Merge branch 'ambient-innovation:main' into mastacheata-patch-template

mastacheata committed:
Update license-checks.yaml

Fix pull license-check.tpl

mastacheata committed:
feat: update readme

update README.md with up-to-date links and instructions

mariusburfey reviewed (approved):
Ein Kommentar, wofür "license-checks.tpl" nötig ist, wäre eine gute Ergänzung finde ich.

(Auch um die Frage zu beantworten, warum es ohne das wget vorher auch schon funktioniert hat.)

mastacheata committed:
Merge branch 'main' into mastacheata-patch-template

mastacheata committed:
Update license-checks.yaml

add comments to explain why the license-check template needs to be downloaded in the first place and what it's used for

mastacheata commented:
> Ein Kommentar, wofür "license-checks.tpl" nötig ist, wäre eine gute Ergänzung finde ich.
> 
> (Auch um die Frage zu beantworten, warum es ohne das wget vorher auch schon funktioniert hat.)

Done @mariusburfey bitte schau nochmal drüber.
